editor/editor_dock_manager.cpp

```diff
@@ -434,6 +507,18 @@ void EditorDockManager::save_docks_to_config(Ref<ConfigFile> p_layout, const Str
 
 	p_layout->set_value(p_section, "dock_floating", floating_docks_dump);
 
+	Array bottom_docks_dump;
+
+	for (Control *bdock : bottom_docks) {
+		Control *dock = bdock;
+
+		String name = dock->get_name();
+
+		bottom_docks_dump.push_back(name);
```

Comment:
```suggestion
		Control *dock = bdock;
		String name = dock->get_name();
		bottom_docks_dump.push_back(name);
```
Too much empty space.